app/views/drag_queens/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>DragQueen#index</h1>
+<p>Find me in app/views/drag_queen/index.html.erb</p>
+
+<% @dragqueen.each do |drag| %>
+  <h2><%= drag.name %></h2>
```

Comment:
Here, I recommend linking to the show page & edit page as well as a delete button, so that your users can navigate to the different parts of your site.
```
<%= link_to drag_queen_path(dragqueen) %> # show page link
<%= link_to edit_drag_queen_path(dragqueen) %> # edit page link
<%= link_to delete_drag_queen_path(dragqueen) %> # delete route link
```